### PR TITLE
Misc. typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3`

### CODE REVIEW

File: src/lib-http/test-http-payload.c [hunk 1]
```diff
@@ -208,7 +208,7 @@ static struct http_server *http_server;
 static struct io *io_listen;
 static struct client *clients;
 
-/* location: /succes */
+/* location: /success */
```
Comment (luzpaz) [inline src/lib-http/test-http-payload.c]: Is this correct ?

File: src/lib-master/master-service.c [hunk 2]
```diff
@@ -307,7 +307,7 @@ master_service_init(const char *name, enum master_service_flags flags,
 			i_fatal(MASTER_CLIENT_LIMIT_ENV" missing");
 		master_service_set_client_limit(service, count);
 
-		/* seve the process limit */
+		/* save the process limit */
```
Comment (luzpaz) [inline src/lib-master/master-service.c]: please review

File: src/lib-smtp/smtp-server-cmd-xclient.c [hunk 3]
```diff
@@ -52,13 +52,13 @@ cmd_xclient_recheck(struct smtp_server_cmd_ctx *cmd,
...
 
-	/* all preceeding commands have finished and now the transaction state is
+	/* all preceding commands have finished and now the transaction state is
 	   clear. This provides the opportunity to re-check the protocol state */
 	if (!cmd_xclient_check_state(cmd))
 		return;
 	smtp_server_connection_set_state(conn, SMTP_SERVER_STATE_XCLIENT);
 
-	/* succes; send greeting */
+	/* success; send greeting */
```
Comment (luzpaz) [inline src/lib-smtp/smtp-server-cmd-xclient.c]: please review

**Comment by luzpaz** [comment]
please review

**Comment by sirainen** [comment]
Merged https://github.com/dovecot/core/commit/a88e4f74ef4934a19eb3440721f752f3bfb74c5f

**Pull request closed without merging**